lib/developer_portal/app/assets/stylesheets/stats/c3.css

```diff
@@ -0,0 +1,167 @@
+/*-- Chart --*/
+.c3 svg {
```

Comment:
These are our styles?

Reply:
right, should be in a `vendor` folder, no?

Reply:
That really depends from where it comes.
Is it from some npm module? Then we should reference that installed module.

Reply:
I don't think so, @didierofrivia ?